web/packages/teleport/src/Assist/Chat/ChatItem/Action/RunAction.tsx

```diff
@@ -245,11 +270,14 @@ function NodeOutput(props: NodeOutputProps) {
         </LoadingContainer>
       )}
 
-      {props.state.stdout && (
-        <NodeContent>
-          <pre>{props.state.stdout}</pre>
-        </NodeContent>
-      )}
+      {props.state.stdout !== undefined &&
+        (props.state.stdout === '' ? (
```

Comment:
So if `stdout` is `undefined` we want to show `<NodeContent>`?

IMO it should be `!props.state.stdout`
It checks for undefined or empty value.

Reply:
Hmm, this works well in practice. Unless I'm misreading the parentheses/ternaries, this is equivalent to:

```js
if (props.state.stdout !== undefined) {
  if (props.state.stdout === '')
    return "Empty output";
  }
  return <NodeOutput>...</NodeOutput>
} else {
  // props.state.stdout === undefined, display nothing
}
```

Reply:
I didn't notice the parentheses 🤦‍♂️ 
Sorry for the noise.